src/components/Tokens/TokenDetails/LoadingTokenDetails.tsx

```diff
@@ -11,20 +13,21 @@ import { DeltaContainer, TokenPrice } from './PriceChart'
 import { StatPair, StatWrapper, TokenStatsSection } from './StatsSection'
 
 const LoadingChartContainer = styled(ChartContainer)`
-  height: 336px;
+  border-bottom: 1px solid ${({ theme }) => theme.backgroundOutline};
+  height: 313px;
   overflow: hidden;
 `
 
 /* Loading state bubbles */
 const LoadingDetailBubble = styled(LoadingBubble)`
-  height: 16px;
+  height: 17px;
   width: 180px;
 `
 const TitleLoadingBubble = styled(LoadingDetailBubble)`
   width: 140px;
 `
 const SquareLoadingBubble = styled(LoadingDetailBubble)`
-  height: 32px;
+  height: 34px;
```

Comment:
same with these, can you just clarify with design.  If this is what they want then I'm good with it